_docs/other/community-geyser-projects.md

```diff
@@ -75,7 +75,7 @@ Plugin that blocks Java players from joining your Geyser server, so you can have
 
 Creator: [Camotoy](https://github.com/Camotoy)
 
-## GeyserPlaceholders
+## ~~GeyserPlaceholders~~(Discontinued)
```

Comment:
I would remove this from the list if it's discontinued

Reply:
ok